src/workerd/io/actor-cache.c++

```diff
@@ -1868,7 +1868,7 @@ kj::OneOf<bool, kj::Promise<bool>> ActorCache::delete_(Key key, WriteOptions opt
     // fulfiller.
     auto paf = kj::newPromiseAndFulfiller<uint>();
     countedDelete->resultFulfiller = kj::mv(paf.fulfiller);
-    KJ_IF_SOME(p, getBackpressure()) {
+    KJ_IF_SOME(p, flushIfNeeded()) {
```

Comment:
Oops, looks like this evaded me because it changed in the counted delete commit. Will fix.